src/main/scala/vsys/blockchain/state/opcdiffs/OpcDiffer.scala

```diff
@@ -56,4 +56,12 @@ object OpcDiffer {
   def apply(context: ExecutionContext)(opc: Array[Byte], data: Seq[DataEntry]): Either[ValidationError, (OpcDiff, Seq[DataEntry])] =
     opc.headOption.flatMap(OpcType.fromByte(_)).toRight(ContractUnsupportedOPC).flatMap(_.opcDiffer.parseBytes(context)(opc.tail, data))
 
+  // res is call-by-name
+  def updateStack(dataStack: Seq[DataEntry], pointer: Byte, res: => Either[ValidationError, DataEntry]): Either[ValidationError, Seq[DataEntry]] =
```

Comment:
@ncying res is a call-by-name parameter which is delayed loading.

Reply:
checked, just reply to u at the same time. ;-)